projects/laji/src/app/shared/service/laji-api.worker.ts

```diff
@@ -45,22 +46,22 @@ function fetchPersonToken(): Observable<string> {
   return token$;
 }
 
-function convertToken(data: string, replace: string, to: string) {
+function convertToken(data: any, replace: string, to: string) {
   if (typeof data === 'string') {
     const rep = new RegExp(replace, 'gm');
     return data.replace(rep, to);
   }
   return data;
 }
 
-function replaceToken(token: string, request: any): any {
+function replaceToken(token: string, request: Record<string, unknown>): any {
   if (typeof request.url === 'string') {
     request.url = convertToken(request.url, PERSON_TOKEN, personToken);
   }
-  if (request.body && typeof request.body === 'object' && typeof request.url === 'string' && request.url.indexOf('graphql') !== -1) {
-    if (request.body.variables) {
+  if (request.body && Util.isObject(request.body) && typeof request.url === 'string' && request.url.indexOf('graphql') !== -1) {
```

Comment:
refact to:    if (isGraphql(request) {

When would request.body not be object?

Reply:
> refact to: if (isGraphql(request) {

Done

> When would request.body not be object?

I don't know :)